Makefile

```diff
@@ -21,4 +21,17 @@ test:
 	go vet ./...
 	go test ./...
 
-.PHONY: rp clean image test
+generate-sdk:
+	go generate ./...
+	# Due to orering https://github.com/Azure/autorest.go/blob/3f9bdd60fd7a7c8740bbe3ec986f583f7ffa5fbd/src/Model/CodeModelGo.cs#L120-L141
+	# package FQDN are either hardcoded to azure sdk or set wrong. For this we do
+	# replace in the end
+	podman run --privileged --workdir=/go/src -it -v $(GOPATH):/go --entrypoint autorest \
```

Comment:
running --privileged and sharing $GOPATH?  No wonder you wiped out your GOPATH.  Let's not do that...